cms/standard_pages/tests/test_models.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from django.utils import timezone
+from django.utils.formats import date_format
+from wagtail.test.utils import WagtailTestUtils
+
+from cms.core.tests.factories import ContactDetailsFactory
+from cms.standard_pages.tests.factories import MethodologyPageFactory
+
+
+class MethodologyPageTestCase(WagtailTestUtils, TestCase):
```

Comment:
Issue: It'd be great to add some tests that the page loads with `IS_EXTERNAL_ENV` set.

Reply:
Added in [13211ed](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/13211edbf34a1a11afe50044d4f602f1a88f5b0c)